packages/jss/src/types/jss.js

```diff
@@ -188,9 +188,9 @@ export type StyleSheetOptions = {|
   Renderer?: Class<Renderer> | null,
   insertionPoint?: InsertionPoint,
   jss: Jss
-|}
+}
```

Comment:
Use explicit inexact type syntax `{value: number, ...}`

Reply:
why? it's by default inexact

Reply:
Not for long. Flow introduced explicit inexact syntax to eventually migrate to exact by default. Btw it can be enabled optionally now.

Reply:
There is also a lint rule
https://flow.org/en/docs/linting/rule-reference/#toc-implicit-inexact-object

Reply:
when they do it we will have to change it everywhere, I take it as a separate task

Reply:
Is it necessary to make it inexact? It does not solve theme problem.

Reply:
sec, trying to fix other places which led to the errors ...

Reply:
changed it everywhere to be exact, had to dig through dozens of functions